fortinet_fortigate/icon_fortinet_fortigate/util/util.py

```diff
@@ -28,7 +28,9 @@ class Helpers(object):
                        " The offended source is temporarily blocked for certain amount of time.",
               "assistance": "Wait out the access timeout."},
         500: {"cause": "Internal Server Error: Internal error when processing the request.",
-              "assistance": "Contact support for assistance."}
+              "assistance": "The most common causes of this error are invalid requests."
```

Comment:
This line is weird because the user doesn't make the request, we do. I wonder if there's something better we can say here.